Resources/doc/reference/installation.rst

```diff
@@ -15,24 +15,39 @@ Installation
     //...
         "sonata-project/classification-bundle": "dev-master",
     //...
+    
+    },
+    "require-dev": {       
+        // ...
+        "sonata-project/easy-extends-bundle": "dev-master"
```

Comment:
Is there a `stable` alias in composer?. The master shouldn't be the prefered version